### PR TITLE
fixes #4625 feat(nimbus): Show entire JEXL expression on audience table

### LINKED ISSUE #4625: Show entire JEXL expression on audience table
Right now we only render the custom part of the targeting expression, it would be helpful when reviewing to be able to see the **complete** targeting expression ahead of requesting review.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/EXP-958)
┆Issue Number: EXP-958

### PR DESCRIPTION
**Pull request by k88hudson**
Because reviewers/QA needs to see the entire JEXL expression when they're reviewing, at least for now in the early stages of development. Moving it to the model makes it easier to render in internal UI v.s.  just in the v6 serializer.

### CODE REVIEW

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 1]
```diff
@@ -24,21 +24,75 @@ def test_str(self):
...
+        self.assertEqual(
+            experiment.targeting,
+            (
+                "version|versionCompare('83.!') >= 0 "
+                "&& localeLanguageCode == 'en' "
+                "&& 'app.shield.optoutstudies.enabled'|preferenceValue"
+            ),
+        )
+
+    def test_serializer_outputs_expected_targeting_for_mobile(self):
```
Comment (jaredlockhart) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:44]: `serializer` doesn't apply here, maybe replace that with the method name?

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 2]
```diff
@@ -24,21 +24,75 @@ def test_str(self):
...
-            targeting_config_slug=NimbusExperiment.TargetingConfig.ALL_ENGLISH
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_83,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.ALL_ENGLISH,
+            application=NimbusExperiment.Application.FENIX,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
+        )
+
+        self.assertEqual(experiment.targeting, "localeLanguageCode == 'en'")
+
+    def test_serializer_outputs_empty_targeting_for_mobile_without_targeting(self):
```
Comment (jaredlockhart) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:54]: same

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 3]
```diff
@@ -24,21 +24,75 @@ def test_str(self):
...
+            NimbusExperiment.Status.ACCEPTED,
+            firefox_min_version=NimbusExperiment.Version.FIREFOX_83,
+            targeting_config_slug=NimbusExperiment.TargetingConfig.NO_TARGETING,
+            application=NimbusExperiment.Application.FENIX,
+            channel=NimbusExperiment.Channel.NO_CHANNEL,
+        )
+
+        self.assertEqual(experiment.targeting, None)
+
+    def test_serializer_outputs_targeting_for_experiment_without_firefox_min_version(
```
Comment (jaredlockhart) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:65]: same

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 4]
```diff
@@ -24,21 +24,75 @@ def test_str(self):
...
+        self.assertEqual(
+            experiment.targeting,
+            (
+                'browserSettings.update.channel == "nightly" '
+                "&& localeLanguageCode == 'en' "
+                "&& 'app.shield.optoutstudies.enabled'|preferenceValue"
+            ),
+        )
+
+    def test_serializer_outputs_targeting_without_channel_version_targeting(self):
```
Comment (jaredlockhart) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:85]: same

**Review by jaredlockhart — APPROVED** [review]
lgtm, maybe just rename those model method test names 👍 Thnx @k88hudson ! 🎉

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py (hunk 1 above)
Comment (k88hudson) [reply app/experimenter/experiments/tests/test_models/test_models_nimbus.py:44]: ah whoops

**Pull request merged**